packages/shopware-6-client/src/services/navigationService.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  getNavigationEndpoint
+} from "../endpoints";
+import { apiService } from "../apiService";
+
+interface NavigationResponse {
+  count: number,
+  elements: NaviationElement[]
+}
+
+interface NaviationElement {
+  id: string
+  path: string
+  name: string
+  children: NaviationElement[] | null,
+  count: number,
+  level: number,
+  extensions: any[]
+}
+
+
+export async function getNavigation(
```

Comment:
always try to take params as an object, then it's easy to extend and change the interface without breaking changes. Also, order of params doesn't matter then.